components/studio/bin/hab-studio.sh

```diff
@@ -83,15 +83,20 @@ SUBCOMMANDS:
     version   Prints version information
 
 ENVIRONMENT VARIABLES:
-    HAB_ORIGIN        Propagates this variable into any studios
-    HAB_ORIGIN_KEYS   Installs secret keys (\`-k' option overrides)
-    HAB_STUDIOS_HOME  Sets a home path for all Studios (default: /hab/studios)
-    HAB_STUDIO_ROOT   Sets a Studio root (\`-r' option overrides)
-    NO_SRC_PATH       If set, do not mount source path (\`-n' flag overrides)
-    QUIET             Prints less output (\`-q' flag overrides)
-    SRC_PATH          Sets the source path (\`-s' option overrides)
-    STUDIO_TYPE       Sets a Studio type when creating (\`-t' option overrides)
-    VERBOSE           Prints more verbose output (\`-v' flag overrides)
+    HAB_NOCOLORING      Disables text coloring mode despite TERM capabilities
+    HAB_NONINTERACTIVE  Disables interactive progress bars despite tty
+    HAB_ORIGIN          Propagates this variable into any studios
+    HAB_ORIGIN_KEYS     Installs secret keys (\`-k' option overrides)
+    HAB_STUDIOS_HOME    Sets a home path for all Studios (default: /hab/studios)
+    HAB_STUDIO_ROOT     Sets a Studio root (\`-r' option overrides)
+    NO_SRC_PATH         If set, do not mount source path (\`-n' flag overrides)
+    QUIET               Prints less output (\`-q' flag overrides)
+    SRC_PATH            Sets the source path (\`-s' option overrides)
+    STUDIO_TYPE         Sets a Studio type when creating (\`-t' option overrides)
+    VERBOSE             Prints more verbose output (\`-v' flag overrides)
+    http_proxy          Sets an http_proxy environment variable inside the Studio
+    https_proxy         Sets an https_proxy environment variable inside the Studio
+    no_proxy            Sets a no_proxy environment variable inside the Studio
```

Comment:
Great to see complete env var docs! We should update https://github.com/habitat-sh/habitat/blob/master/www/source/docs/reference/environment-vars.html.md as well.